department-of-veterans-affairs/caseflow
# seed data for feature to restrict poa visibility

Resolves [Jira ticket 1522](https://vajira.max.gov/browse/CASEFLOW-1522) and is tied to PR #16433. 

### Description
This PR
- creates seed data to demo the work in #16433
- puts the work from #16433 behind a feature toggle

### Acceptance Criteria
- [ ] Seed data exists

### Testing Plan
1. Repeat the testing plan from 16433 and confirm that the feature toggle has hidden that PR's changes
2. Look up the veteran with the file number `42_424_242 ` and confirm it has an appeal with a decision issue whose decision date is in the future.
3. Confirm you can run `make seed` successfully with the new seed data.
